docs/Advanced Features/workflows.md

```diff
@@ -143,7 +143,7 @@ and specify `workflow=None` in the API:
 Package test/package@06b2815 pushed to s3://quilt-sergey-dev-metadata
 ```
 
-Also `default_workflow` can be set in config to specify which workflow will be used
+Also `default_workflow` can be set in the config to specify which workflow will be used
```

Comment:
```suggestion
You can also specify a `default_workflow` in `config.yml`.
```

Reply:
I left this up to you, but generally I'd like to be less specific here (we already specified path in the beginning) and do not mention `config.yml` again since we could change the path in the future or maybe even move this configuration out of S3.